pandunia-lekse.csv

```diff
@@ -1757,6 +1764,9 @@ lefti||||left hand||izquierdo|||||||||||||||lewy (lewostronny, z lewej strony)
 leftiste||||leftist (left-winger)||izquerdista||||||||||||||vasemmistolainen|lewicowiec (lewak)
 legi|||eng:light, spa:ligero, fra:léger, rus:лёгкий (l’ogkiy), pol:lekki, ben:লঘু (laghu)|light (weightless)|léger|ligero|||||||||||||||
 legos||||Lagos|||||||||||||||||
+leka||||amuse (entertain)||divertir (entretener)|||||||||||||amuzi|viihdyttää (huvittaa)|bawić, zabawiać
+leke|||zho:乐 (lè), yue:樂 (lok6), wuu:樂 (loq5), jpn:楽 (raku), kor:즐길 락 (jeulgil rak)|fun (amusement)||diversión|||||||||||||amuzo|huvi (hauskuus)|zabawa, rozrywka
```

Comment:
This is not the right word in my opinion because 樂 means happiness and joy, like Pandunia's "suk-".

How about "wan"? I can make room for it by changing the current wan- to vol-. We have to do it anyway if we shall merge <w> into <v>. See https://en.wiktionary.org/wiki/%E7%8E%A9

Reply:
That works.

Reply:
actually, are you sure about **vola**? it's not really used in Spanish or Portuguese, and in English "volition" means "will", not "want", so it's really just French and Italian at this point. what about **desira**, or **yuka** (from 欲, would require changing "jade" to **giuke**)?